pandas/tests/groupby/test_categorical.py

```diff
@@ -48,8 +48,9 @@ def setUp(self):
              'D': np.array(
                  np.random.randn(8), dtype='float32')})
 
-        index = MultiIndex(levels=[['foo', 'bar', 'baz', 'qux'], ['one', 'two',
-                                                                  'three']],
+        index = MultiIndex(levels=[pd.CategoricalIndex(
```

Comment:
no, ``.mframe`` is used by several tests, create a new df local to the test, this is very specific.